### DISCUSSION TITLE
Using hostnames vs IP address in 0.6.9

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#530] [Q&A] [answered]
I have juts moved to the v0.6.9 test release and I noticed that in the works page all hostnames are juts being reported as the first octet of the IP address.

I use the physical IP address of each of each node as the "hostname" of the node.

This used to display properly in version v0.6.7  (except for the chia full node) but seems to have changed.

This also effects the reporting in the summary tab as it looks like there is only 1 harvester with the node name "192"

Questions:
1. Does Machinaris require a hostname for each node?
2. Does this host name need to be resolved by DNS?
3. If the hostname is not resolved by DNS does it fail back to the statically entered "worker ip"

It seems everything is functional.. juts not displaying correctly from what I can tell.

See attached images

![Challenge-Capture](https://user-images.githubusercontent.com/95300898/148106168-d1d44934-4dc5-499c-b63d-e999f61dd218.JPG)

![Workers-Capture](https://user-images.githubusercontent.com/95300898/148106197-1b83e2d1-a979-488f-9a04-7ce78428dd78.JPG)

**Comment by guydavis** [comment, marked as answer]
Hi, first off... thanks for testing the `:test` image stream.

Please use a shortname for your `hostname` value, whether in `docker-compose.yml` or in the Unraid "Extra Parameters" section of the Docker configuraton.  Please leave the IP address in the `worker_address` and `controller_host` attributes instead.

The short hostname is just for display (as in your screenshots).  The short hostname does not need to resolve on your LAN, that is what the IP address in `worker_address` is used for.  Hope this helps.

**Comment by LeroyINC** [comment]
that seemed to do the trick... thanks

**Reply by guydavis** [reply]
Great! Apologies for the change.  I am working to better support TrueNAS deployments which have a particular hostname format.  Thanks for the understanding on making this configuration change.
